### PR TITLE
Added FAQ section 

### LINKED ISSUE #326: Creating a FAQ section
I want to create a FAQ section which contains frequently asked questions regarding Algorithm

### PR DESCRIPTION
**Pull request by harshb02**
# Description
Added a FAQ section in the index.html. I have written dummy text for now which will be replaced by the content as soon I get it.

Fixes #326 

## List any dependencies that are required for this change 
NA

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Ran on local server

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :

    - After :
    

demo6

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,6 +216,107 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+<!--FAQ Section Starts-->
+
+<section class="fnq" id="fnq">
+    <div class="container mt-5 mb-5">
+        <h1 class="text-center">FREQUENTLY ASKED QUESTIONS</h1>
+        <hr>
+        <br><br>
+    <div class="row row-content">
+          <div class="col-12">
+           <div class="accordion" id="accordionExample">
```
Comment (harikesh409) [inline index.html:228]: Make the id more meaningful

File: index.html [hunk 2]
```diff
@@ -216,6 +216,107 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
 <!--Section-Sharpen your skills ends-->
 
+<!--FAQ Section Starts-->
+
+<section class="fnq" id="fnq">
+    <div class="container mt-5 mb-5">
+        <h1 class="text-center">FREQUENTLY ASKED QUESTIONS</h1>
+        <hr>
+        <br><br>
+    <div class="row row-content">
```
Comment (harikesh409) [inline index.html:226]: `row-content` is unnecessary tag.

File: index.html [hunk 3]
```diff
@@ -216,6 +216,107 @@ <h1 class="text-center">Sharpen Your Skills</h1>
 
 <!--Section-Sharpen your skills ends-->
 
+<!--FAQ Section Starts-->
+
+<section class="fnq" id="fnq">
+    <div class="container mt-5 mb-5">
+        <h1 class="text-center">FREQUENTLY ASKED QUESTIONS</h1>
```
Comment (harikesh409) [inline index.html:223]: Make it init caps instead of all caps.

Comment (harikesh409) [reply index.html:223]: Add other styles also which are present for other headings.

File: css/style.css [hunk 4]
```diff
@@ -345,6 +345,28 @@ background-color: #FFF;
...
+    margin-bottom: 0;
+    background-color: rgba(0,0,0,.03);
+    box-shadow: 0px 10px 40px rgba(0,0,0,0.4);
+    border-bottom: 1px solid rgba(0,0,0,.125);
+}
+
+.fnq a {
+    font-size: 1.5rem;
+    font-weight: bold;
+    text-transform: uppercase;
```
Comment (harikesh409) [inline css/style.css:361]: Transforming to uppercase is not needed.

File: css/style.css [hunk 5]
```diff
@@ -345,6 +345,28 @@ background-color: #FFF;
     }
 /*********Section-Sharpen your skills ends**********/
 
+/*********FAQ section starts**********/
+
+.fnq .card-header {
+    padding: .75rem 1.25rem;
```
Comment (harikesh409) [inline css/style.css:351]: Except `box-shadow` rest all are default properties that comes from bootstrap itself so they can be removed.

File: index.html [hunk 6]
```diff
@@ -216,6 +216,107 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+        <hr>
+        <br><br>
+    <div class="row row-content">
+          <div class="col-12">
+           <div class="accordion" id="accordionExample">
+      <div class="card">
+        <div class="card-header" id="headingOne">
+          <h5 class="mb-0">
+            <a data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne" style="color: black">
+                <i class="fas fa-plus"></i>  Services we provide you
```
Comment (harikesh409) [inline index.html:233]: Just a suggestion.
It looks good if the plus changes to minus on click.

File: index.html [hunk 7]
```diff
@@ -216,6 +216,107 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+          </h5>
+        </div>
+        <div id="collapseSix" class="collapse" aria-labelledby="headingSix" data-parent="#accordionExample">
+          <div class="card-body">
+            Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
+          </div>
+        </div>
+      </div>
+    </div>
+            </div>
```
Comment (harikesh409) [inline index.html:314]: Maintain proper indentations.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harshb02** [comment]
I have made the changes kindly review them @harikesh409

File: index.html [hunk 8]
```diff
@@ -218,104 +218,110 @@ <h1 class="text-center">Sharpen Your Skills</h1>
 
 <!--FAQ Section Starts-->
 
-<section class="fnq" id="fnq">
+<div class="fnq" id="fnq">
```
Comment (harikesh409) [inline index.html:221]: `section` is a more preferable tag for SEO.

File: index.html [hunk 9]
```diff
@@ -218,104 +218,110 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
     <div class="container mt-5 mb-5">
```
Comment (harikesh409) [inline index.html:222]: Instead of two classes mt-5 and mb-5 you can use my-5

File: index.html [hunk 10]
```diff
@@ -218,104 +218,110 @@ <h1 class="text-center">Sharpen Your Skills</h1>
...
+                             Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
+                        </div>
+                    </div>
+                </div>
             </div>
-          </div>
         </div>
     </div>
-    </section>
+</div>
```
Comment (harikesh409) [inline index.html:322]: Check indentations.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harshb02** [comment]
Implemented the changes. Kindly review them @harikesh409

**Review by harikesh409 — APPROVED** [review]

**Review by ravikr126 — APPROVED** [review]
Done

**Pull request merged**